WinNSMenu.m

```diff
@@ -271,11 +191,15 @@ HMENU r_build_menu_for_itemmap(NSMenu *menu, BOOL asPopUp, BOOL fakeItem, NSMapT
 	{
 	  flags = MF_STRING;
 	  s = menu_tag++;
-	  if(fakeItem)
```

Comment:
@gcasa @fredkiefer These lines were actually causing GSFakeMenuItems to replace NSMenuItems in memory leading to crashes when you clicked on a pop up.